lib/rihanna/job.ex

```diff
@@ -8,7 +8,9 @@ defmodule Rihanna.Job do
 
   @callback perform(arg :: any) :: :ok | {:ok, result} | :error | {:error, reason}
   @callback after_error({:error, reason} | :error | Exception.t(), arg) :: any()
-  @optional_callbacks after_error: 2
+  @callback retry_at({:error, reason} | :error | Exception.t(), arg, pos_integer) ::
```

Comment:
Where is the documentation?

Reply:
Not added it yet. Wanted to confirm you approved of the implementation.